### PR TITLE
Add readthedocs documentation.

### PR DESCRIPTION
**Pull request by smarlowucf**
Cleanup readme and update requirements for sphinx docs build.

Readthedocs project setup and will build to https://mash-client.readthedocs.io/en/latest/ once merged.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ $ git clone https://github.com/SUSE-enceladus/mash-client.git
 $ cd mash-client
 
 # Activate virtual Environment then install
-# ipa and dev dependences in editable mode
+# mash client and dev dependences in editable mode
```
Comment (rjschwei) [inline CONTRIBUTING.md]: s/dependences/dependencies/

File: docs/source/conf.py [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'MASH Client'
+copyright = '2019, SUSE LLC. All rights reserved'
+author = 'SUSE'
```
Comment (rjschwei) [inline docs/source/conf.py]: s/SUSE/SUSE Public Cloud Team/

File: docs/source/config.rst [hunk 3]
```diff
@@ -0,0 +1,33 @@
+Configuration
+=============
+
+MASH Client uses a YAML config file. By default the client will look
```
Comment (rjschwei) [inline docs/source/config.rst]: s/config/configuration/

"Which is expected with the path *~/.config/mash_client/config.yaml*"

File: docs/source/config.rst [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+This location can be configured with each command using the *-C/--config*
+option. For example::
+
+    mash --config ~/new/config.yaml job add ...
+
+Options
+-------
+
+The following options are currently available in the config file:
```
Comment (rjschwei) [inline docs/source/config.rst]: s/config/configuration/

File: docs/source/install.rst [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+openSUSE package
+----------------
+
+Perform the following commands as root:
+
+.. code-block:: console
+
+   $ zypper ar http://download.opensuse.org/repositories/Cloud:/Tools/<distribution>
+   $ zypper refresh
```
Comment (rjschwei) [inline docs/source/install.rst]: the refresh can be skipped but doesn't hurt

**Comment by smarlowucf** [comment]
@rjschwei Cleaned up per comments.

**Pull request merged**